gtk/src/toga_gtk/container.py

```diff
@@ -156,17 +156,20 @@ def do_size_allocate(self, allocation):
         computed based on this new available size, and that new geometry will be applied
         to all child widgets of the container.
         """
-        # print(self._content, f"Container layout {allocation.width}x{allocation.height} @ {allocation.x}x{allocation.y}")  # noqa: E501
+        # print(self._content, f"Container layout "
+        # "{allocation.width}x{allocation.height} @ {allocation.x}x{allocation.y}")
+        # # noqa: E501
```

Comment:
Even though it's commented out, this line can still be split like any other string.

Reply:
ah, the noqa pragma was existing, and of course should have been removed.

Reply:
As a general rule, we'd like to avoid noqa lines *at all*. We'll include them if they're absolutely required, but there's a pretty high bar for that. Even if they're pre-existing, it's worth re-evaluating whether there's a way to remove them.